jsonrpsee/Cargo.toml

```diff
@@ -34,9 +35,11 @@ ws-client = ["jsonrpsee-ws-client", "jsonrpsee-types", "jsonrpsee-core"]
 ws-server = ["jsonrpsee-ws-server", "jsonrpsee-types", "jsonrpsee-core"]
 macros = ["jsonrpsee-proc-macros", "jsonrpsee-types", "jsonrpsee-core/client", "tracing"]
 
-client = ["http-client", "ws-client", "wasm-client"]
-server = ["http-server", "ws-server"]
-full = ["client", "server", "macros", "async-client", "client-ws-transport"]
+client-full = ["client", "http-client", "ws-client", "wasm-client", "client-ws-transport", "client-web-transport", "async-client"]
+client = ["macros"]
```

Comment:
I guess we could rename this to client-core as well and keep the client feature, not sure what's the best

Reply:
I think I prefer "client" being kept as-is and "client-core" for those "optimisation" cases where you want less. Also it wouldn't be a breaking change I guess that way so it's a win win really :)

Reply:
fair enough, I'm convinced